File: leetcode/2798/Solution.java
class Solution {
    public int numberOfEmployeesWhoMetTarget(int[] hours, int target) {
        int ans = 0;
        for (int hour : hours) {
            if (hour >= target) {
                ans++;
            }
        }
        return ans;
    }

    public static void main(String[] args) {
        Solution sol = new Solution();
        int[] hours = {0,1,2,3,4};
        int target = 2;
        System.out.println("test: " + sol.numberOfEmployeesWhoMetTarget(hours, target));
    }
}

File: leetcode/2798/Solution.py
from typing import List

class Solution:
    def numberOfEmployeesWhoMetTarget(self, hours: List[int], target: int) -> int:
        return sum(h >= target for h in hours)

if __name__ == '__main__':
    sol = Solution()
    hours = [0,1,2,3,4]
    target = 2
    print("test:", sol.numberOfEmployeesWhoMetTarget(hours, target))